clients/src/main/java/org/apache/kafka/clients/producer/internals/BufferPool.java

```diff
@@ -43,6 +43,8 @@
  */
 public class BufferPool {
 
+    static final String WAIT_TIME_SENSOR_NAME = "bufferpool-wait-time";
```

Comment:
I am not sure it is useful to have a static variable for this string since it is used only once. It is also different from the code style in `Sender.SenderMetrics` which assigns sensor name directly. Maybe we can preserve the existing style for simplicity and code style consistence.